packages/cli/odahuflow/cli/parsers/local/training.py

```diff
@@ -147,7 +147,7 @@ def run(client: ModelTrainingClient, train_id: str, manifest_file: List[str], ma
         click.echo(f'{train_id} training not found. Trying to retrieve it from API server')
         mt = client.get(train_id)
 
-    toolchain = toolchains.get(mt.spec.toolchain)
+    toolchain = toolchains.get(mt.spec.toolchain) or ""
```

Comment:
To be honest, I don't see a lot of sense in this change. Why do we need `toolchain` variable to be a string? Currently it's implicit type (basing on the code) is `Optional[ToolchainIntegration]`. If `toolchains.get(mt.spec.toolchain)` cannot find such key it returns `None` which makes more sense than an empty string. 
I would leave it as it was.

Reply:
I just tried to improve output when toolchain not found.

**before:** 
![image](https://user-images.githubusercontent.com/49116723/97697277-73d6fb80-1aaf-11eb-99ac-837e78af2154.png)
**after:**
![image](https://user-images.githubusercontent.com/49116723/97697342-8bae7f80-1aaf-11eb-8ede-8714e3e4d631.png)

Reply:
@BPylypenko  Hmm.. Basically, this is the case when toolchain is missing, which is an invalid request. As far as I see in the code, in this case we will just fail on retrieving `None` toolchain from API. It will return 404. 
So current behavior is already pretty much self-explaining. If you want to make this message prettier, maybe insert an `if` before line 150:
```
if mt.spec.toolchain is None:
    raise ValueError(f'Toolchain cannot be None in Model Training')
```